channels_redis/core.py

```diff
@@ -544,7 +560,11 @@ async def new_channel(self, prefix="specific"):
         Returns a new channel name that can be used by something in our
         process as a specific channel.
         """
-        return "%s.%s!%s" % (prefix, self.client_prefix, uuid.uuid4().hex,)
+        return "%s.%s!%s" % (
```

Comment:
@carltongibson I changed this because `black` was complaining about it

Reply:
No problem. Whatever black says. 😀